lib/http/preprocessors.js

```diff
@@ -114,7 +114,10 @@ const authHandler = ({ Sessions, Users, Auth }, context) => {
 
     // actually try to authenticate with it. no Problem on failure. short circuit
     // out if we have a GET or HEAD request.
```

Comment:
I feel like these comments belong above `authBySessionToken()` and the `if` about GET and HEAD.

Reply:
Moved :+1: